.github/workflows/publish-release.yml

```diff
@@ -85,41 +86,41 @@ jobs:
         replace: " "
         include: "**pubspec.yaml"
 
-    - name: Package common
+    - name: Package realm_common
       run: |
-        cp -Lr ../common .
-        cd common
+        cp -Lr ../package/realm_common .
+        cd realm_common
         dart pub publish --dry-run || true
       working-directory: release
 
-    - name: Package generator
+    - name: Package realm_generator
       run: |
-        cp -Lr ../generator .
-        cd generator
+        cp -Lr ../packages/realm_generator .
+        cd realm_generator
         dart pub publish --dry-run || true
       working-directory: release
 
     # realm_flutter has symlinks to native binaries which should not be packaged
     - name: Cleanup binary symlinks
       run: |
-        rm -rf flutter/realm_flutter/android/src/main/cpp/lib
-        rm flutter/realm_flutter/ios/src
-        rm flutter/realm_flutter/ios/realm_dart.xcframework
-        rm flutter/realm_flutter/linux/binary
-        rm flutter/realm_flutter/windows/binary
-        rm example/binary
+        rm -rf packages/realm/android/src/main/cpp/lib
+        rm packages/realm/ios/src
+        rm packages/realm/ios/realm_dart.xcframework
+        rm packages/realm/linux/binary
+        rm packages/realm/windows/binary
+        rm packages/realm/example/binary
 
     - name: Package realm (flutter)
       run: |
-        cp -Lr ../flutter/realm_flutter realm
+        cp -Lr ../packages/realm realm
         cd realm
         flutter pub publish --dry-run || true
       working-directory: release
 
     - name: Package realm_dart
       run: |
         mkdir realm_dart
-        cp -Lr ../bin ../example ../lib ../test realm_dart
+        cp -Lr ../package/realm_dart/bin ../package/realm_dart/example ../package/realm_dart/lib ../package/realm_dart/test realm_dart
```

Comment:
Do we need this convoluted copying now that `realm_dart` is in a dedicated folder?

Reply:
No, I decided to split the branch to reduce the size of the PR, so it ended up being  addressed in the follow up PR #1516. 

Actually I'm wondering why we even store packages in the release tarball and upload to s3? They are available for perpetuity on pub.dev?